### ISSUE TITLE
[1.12.0-RC1] Failed to initialize CoreCLR (seccomp regression)

### ISSUE THREAD

**Issue by samoht** [docker/docker#23587]
**Output of `docker version`:**

```
Client:
 Version:      1.12.0-rc1
 API version:  1.24
 Go version:   go1.6.2
 Git commit:   1f136c1
 Built:        Wed Jun 15 05:11:00 2016
 OS/Arch:      linux/amd64
 Experimental: true

Server:
 Version:      1.12.0-rc1
 API version:  1.24
 Go version:   go1.6.2
 Git commit:   1f136c1
 Built:        Wed Jun 15 05:11:00 2016
 OS/Arch:      linux/amd64
```

**Output of `docker info`:**

```
Containers: 3
 Running: 0
 Paused: 0
 Stopped: 3
Images: 69
Server Version: 1.12.0-rc1
Storage Driver: aufs
 Root Dir: /var/lib/docker/aufs
 Backing Filesystem: extfs
 Dirs: 113
 Dirperm1 Supported: true
Logging Driver: json-file
Cgroup Driver: cgroupfs
Plugins:
 Volume: local
 Network: null host overlay bridge
Swarm: inactive
Runtimes: default
Default Runtime: default
Kernel Version: 4.2.0-36-generic
Operating System: Ubuntu 15.10
OSType: linux
Architecture: x86_64
CPUs: 4
Total Memory: 15.56 GiB
Name: emu
ID: LKRP:5O4R:KBMJ:MAMD:BDVP:RUTH:SDKP:UNOZ:G44Q:3H3E:HMZ3:5LTH
Docker Root Dir: /var/lib/docker
Debug Mode (client): false
Debug Mode (server): false
Registry: https://index.docker.io/v1/
WARNING: No swap limit support
Experimental: true
Insecure Registries:
 127.0.0.0/8
```

**Steps to reproduce the issue:**

```
$ cat Dockerfile
FROM microsoft/dotnet:1.0.0-preview1
ADD project.json .
RUN dotnet restore project.json

$ cat project.json
{
    "name": "Worker",
    "buildOptions": {
        "emitEntryPoint": true,
        "warningsAsErrors": true
    },
    "dependencies": {
        "StackExchange.Redis": "1.1.604-alpha",
        "Npgsql": "3.1.3",
        "Newtonsoft.Json": "9.0.1-beta1",
        "Microsoft.NETCore.App": {
            "type": "platform",
            "version": "1.0.0-rc2-3002702"
        }
    },
    "frameworks": {
        "netcoreapp1.0": { }
    },
    "runtimeOptions": {
        "configProperties": {
            "System.GC.Server": true
        }
    }
}
```

**Describe the results you received:**

```
$ docker build .
Sending build context to Docker daemon 4.608 kB
Step 1 : FROM microsoft/dotnet:1.0.0-preview1
 ---> 1707bb9a2d73
Step 2 : ADD project.json .
 ---> Using cache
 ---> 3fbaf0ff1173
Step 3 : RUN dotnet restore project.json
 ---> Running in 92dee24d170c
Failed to initialize CoreCLR, HRESULT: 0x8007001F
The command '/bin/sh -c dotnet restore project.json' returned a non-zero code: 137
```

**Describe the results you expected:**

docker build does not fail.

**Comment by samoht** [comment]
Running with 1.12-rc1 (non-experimental) I also got:

```
$ docker build .
Sending build context to Docker daemon 4.608 kB
Step 1 : FROM microsoft/dotnet:1.0.0-preview1
1.0.0-preview1: Pulling from microsoft/dotnet
5c90d4a2d1a8: Pull complete
ab30c63719b1: Pull complete
c6072700a242: Pull complete
2a65fccac866: Pull complete
973b97c12bc6: Pull complete
Digest: sha256:c0ed658db4baf6ee6e455b5059d81448b507ed0344d101e1187c894a1b6cea13
Status: Downloaded newer image for microsoft/dotnet:1.0.0-preview1
 ---> 1707bb9a2d73
Step 2 : ADD project.json .
 ---> 3fbaf0ff1173
Removing intermediate container 43f50a3be052
Step 3 : RUN dotnet restore project.json
 ---> Running in ae78895fe50a
failed to create endpoint hungry_hypatia on network bridge: network e8114286baa69774dae8cb442252c0636344bacfb16464ca16e8c56662fc07a0 does not exist
```

Not sure if this is expected. Seems to correct itself after a few resets ...

**Comment by icecrime** [comment]
Cc @mrjana @mavenugo!

**Comment by justincormack** [comment]
The `Failed to initialize CoreCLR, HRESULT: 0x8007001F` is a seccomp regression, my fault, doing a patch.

**Comment by justincormack** [comment]
Fixed by #23593
